source/common/statistic_impl.cc

```diff
@@ -334,4 +342,12 @@ void SinkableCircllhistStatistic::recordValue(uint64_t value) {
   scope_.deliverHistogramToSinks(*this, value);
 }
 
+std::string SinkableCircllhistStatistic::tagExtractedName() const {
+  if (worker_id().has_value()) {
+    return fmt::format("{}.", worker_id().value()) + id();
```

Comment:
Same fmt::format question as in SinkableHdrStatistic

Reply:
Done